config/initializers/devise.rb

```diff
@@ -12,7 +12,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'no-reply@dcabortionfund.org'
+  config.mailer_sender = "no-reply@#{FUND_DOMAIN}" || 'no-reply@dcabortionfund.org'
```

Comment:
oh, oops, my fault! This one's going to break CI, because of how initializers load (in alphabetical order. ask me what deep, evil hole I had to go down to learn this.)

Recommend setting it to `no-reply@#{ENV['FUND_MAILER_DOMAIN']}` instead. A little dirty, but sometimes you can't fight city hall or whatever.